webpack/js/main.js

```diff
@@ -32,6 +32,13 @@ $(document).ready(function () {
             $(".navbar-item.has-dropdown.is-active").removeClass('is-active');
         }
     });
+
```

Comment:
You should also add a sentence to add the class when the page is loaded and if there's a hash in the url